tools/src/test/java/org/apache/kafka/tools/FeatureCommandTest.java

```diff
@@ -67,8 +67,10 @@ public void testDescribeWithKRaft(ClusterInstance cluster) {
         List<String> features = Arrays.stream(commandOutput.split("\n")).sorted().collect(Collectors.toList());
 
         // Change expected message to reflect latest MetadataVersion (SupportedMaxVersion increases when adding a new version)
+        assertEquals("Feature: kraft.version\tSupportedMinVersion: 0\t" +
+                "SupportedMaxVersion: 1\tFinalizedVersionLevel: 0\t", outputWithoutEpoch(features.get(0)));
```

Comment:
How does this test work? Do we explicitly set the FinalizedVersionLevel for kraft.version to 0 somewhere?

Reply:
It's set in `BrokerFeatures.scala`